drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
```

Comment:
@cptvitamin Quick question for you about this drag and drop block: Once a draggable is dropped onto a drop zone, the draggable cannot be interacted with but will still display useful information (e.g. its `<span class="sr">` text will say "The 'circle' item, dropped onto the 'round shapes' zone".

Since it cannot be interacted with at that point, should it have `tabindex=-1`? Or should it have a tabindex since it represents a UI element that contains useful information and was previously interactive?